loadtest/control/simulcontroller/controller.go

```diff
@@ -234,6 +234,18 @@ func (c *SimulController) Run() {
 			run:       c.getInsights,
 			frequency: 0.011,
 		},
+		{
+			run:       c.upsertDraft,
+			frequency: 0.615,
+		},
+		{
+			run:       c.getDrafts,
+			frequency: 0.555,
+		},
+		{
+			run:       c.deleteDraft,
+			frequency: 1.5,
+		},
```

Comment:
Are all of these calculated from community metrics?

Reply:
Also, can we use the `minServerVersion` field in these actions? We should add that from now on to all new actions, and I want to start adding them to the current ones as well.